spec/controllers/two_factor_authentication/otp_verification_controller_spec.rb

```diff
@@ -474,6 +522,21 @@
             with(Analytics::MULTI_FACTOR_AUTH, properties)
         end
       end
+
+      context 'when the user chooses to remember their device' do
```

Comment:
Does the UI allow the user to remember their device in the IdV context? If not, do we still need this test? If we just want to make sure someone can't make a POST request to remember a device (with cURL for example), maybe we should rename the test?

Reply:
:+1: